lib/rpmsg/rpmsg_virtio.c

```diff
@@ -273,13 +273,14 @@ static int rpmsg_virtio_wait_remote_ready(struct rpmsg_virtio_device *rvdev)
 /**
  * @internal
  *
- * @brief Returns buffer size available for sending messages.
+ * @brief Returns buffer size available for sending/receiving messages.
  *
  * @param rvdev	Pointer to rpmsg device
+ * @param tx	Tx or Rx buffer
  *
  * @return Buffer size
  */
-static int _rpmsg_virtio_get_buffer_size(struct rpmsg_virtio_device *rvdev)
+static int _rpmsg_virtio_get_buffer_size(struct rpmsg_virtio_device *rvdev, bool tx)
```

Comment:
please declare an enum instead of using the bool. It will be more explicit

Reply:
Thanks, done